Jenkinsfile

```diff
@@ -3,6 +3,28 @@
 // Jenkins pipeline
 // See documents at https://jenkins.io/doc/book/pipeline/jenkinsfile/
 
+// Docker env used for testing
+// Different image may have different version tag
+// because some of them are more stable than anoter.
+//
+// Docker images are maintained by PMC, cached in dockerhub
+// and remains relatively stable over the time.
+// Flow for upgrading docker env(need commiter)
+//
+// - Send PR to upgrade build script in the repo
+// - Build the new docker image
+// - Tag the docker image with a new version and push to tvmai
+// - Update the version in the Jenkinsfile, send a PR
+// - Fix any issues wrt to the new image version in the PR
+// - Merge the PR and now we are in new version
+// - Tag the new version as the lates
+// - Periodically cleanup the old versions on local workers
+//
+ci_lint = "tvmai/ci-lint:v0.50"
+ci_gpu = "tvmai/ci-gpu:v0.50"
+ci_cpu = "tvmai/ci-cpu:v0.41"
```

Comment:
@tqchen Why do you use `0.41` here for CPU?

Reply:
It was an old tag that we used, and I end up have some problem in upgrading the rust rt, will update once we fixed that problem

Reply:
ok. got it.